worldedit-core/src/main/java/com/sk89q/worldedit/command/SchematicCommands.java

```diff
@@ -205,6 +211,39 @@ public void save(Actor actor, LocalSession session,
                 .buildAndExec(worldEdit.getExecutorService());
     }
 
+    @Command(
+        name = "share",
+        desc = "Share a schematic into your clipboard"
```

Comment:
smh you corrected the `save` and then made the same mistake here
```suggestion
        desc = "Share your clipboard as a schematic online"
```